app.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const { getCategories } = require("./controller");
+
+const app = express();
+
+app.use(express.json());
+
+app.get("/api/categories", getCategories);
+
+app.use("*", (req, res, next) => {
+  res.status(404).send({ msg: "Invalid Path" });
+  next(err);
+});
+
+app.use((err, req, res, next) => {
+  res.status(500).send({ message: "this is a server problem" });
```

Comment:
You want to put a console.log of the error here before you send the 500 so you can see what the error is if any!